media-tv/plex-media-server/plex-media-server-1.23.1.4571.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd unpacker
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+MY_COMMIT="6119e8eed"
+MY_PV="${PV}-${MY_COMMIT}"
+
+MY_URI="https://downloads.plex.tv/plex-media-server-new"
+SRC_URI="
+	amd64? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_amd64.deb )
+	x86? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist strip"
+IUSE="system-openssl"
+
+RDEPEND="
+	acct-group/plex
+	acct-user/plex
+	system-openssl? ( dev-libs/openssl )
+"
+
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=(
+	"usr/lib/plexmediaserver/lib/.*"
+	"usr/lib/plexmediaserver/Resources/Python/lib/python2.7/.*"
+	"usr/lib/plexmediaserver/Resources/Python/lib/python2.7/lib-dynload/_hashlib.so"
+)
+
+S="${WORKDIR}"
```

Comment:
I have same nitpick as sam here, not a rule but I prefer this below `SRC_URI` (without blank line) to match skel.ebuild